completions/bash/multipass

```diff
@@ -90,8 +160,30 @@ _multipass_complete()
                 _filedir
                 return
             ;;
-            # TODO: add "--network" here
+            "--network")
+                compopt -o nospace
+                # TODO: Do use spaces after the completion of an interface name.
+                _multipass_networks
+                opts="${opts} id= mode= mac="
         esac
+
+    # We need to complete the option pairs in two steps. One, when the last input char was an equals sign. Two,
+    # when the user starts to input something, because prev changes when she starts to type a value.
+    elif [[ "${cur}" == '=' ]] && [[ "${prev2}" == ',' || "${prev2}" == '--network' ]] ; then
+        compopt -o nospace
+        _remove_trailing_char
+        _complete_network_var "${prev}"
+
+    elif [[ "${prev}" == '=' ]]; then
+        compopt -o nospace
+        _complete_network_var "${prev2}"
+
+    # Complete now in the case on which there is a comma separating var=val pairs.
+    elif [[ "${cur}" == ',' && "${prev2}" == '=' ]] || [[ "${prev}" == ',' && "${COMP_WORDS[COMP_CWORD-3]}" == '=' ]];
```

Comment:
Another similar case of assuming `,` belongs to --network. Here it gives wrong results in very specific conditions, like `multipass set client.gui.hotkey=ctrl+,[TAB]`.

Reply:
Yep, will fix.

Reply:
Fixed.